finx_multibanking_and_pfm_ongoing_access.postman_collection_v2_1.json

```diff
@@ -62,7 +62,7 @@
 						],
 						"body": {
 							"mode": "raw",
-							"raw": "{\n    \"state\": \"test123\",\n    \"redirect_uri\": \"http://localhost\",\n    \"language\": \"en\",\n    \"allow_multi_selection\": true,\n    \"sync_period\": 90,\n    \"save_secrets\": false,\n    \"user_id\": \"user@test.de\"\n}",
+							"raw": "{\n    \"state\": \"test123\",\n    \"redirect_uri\": \"http://localhost\",\n    \"language\": \"en\",\n    \"allow_multi_selection\": true,\n    \"sync_period\": 90,\n    \"save_secrets\": false,\n    \"user_id\": \"user@qwist.com\"\n}",
```

Comment:
If all users of this collection use the same `user_id`, they are actually sharing the same API user under the hood. I don't think this is the expected behavior. Do we want to generate a unique one? Something like `{uuid}@example.com`?